src/fragments/start/getting-started/next/setup.mdx

```diff
@@ -1,9 +1,13 @@
+<Callout>
+Amplify JavaScript works best with the Next.js <a href="https://nextjs.org/docs/pages/building-your-application">Pages Router</a>. Full support for the App Router introduced in Next.js 13.4 is coming soon.
+</Callout>
+
 To set up the project, you'll first create a new Next.js app with [Create Next App](https://nextjs.org/docs/api-reference/create-next-app), a simple CLI tool that enables you to quickly start building a new Next.js application, with everything set up for you. You'll then add Amplify and initialize a new project.
 
 From your projects directory, run the following commands:
 
 ```bash
-npx create-next-app@latest next-amplified
+npx create-next-app@latest next-amplified --no-app
```

Comment:
This is the primary change which ensures the `create-next-app` will generate the app in the `pages` directory instead of the default `app` directory.